docs/dev/connectors/streamfile_sink.md

```diff
@@ -24,7 +24,7 @@ under the License.
 -->
 
 This connector provides a Sink that writes partitioned files to filesystems
-supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems.html).
+supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems).
```

Comment:
```suggestion
supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems/index.html).
```

Reply:
Hi zentol,
I have been working on translating this doc to Chinese. So, I will fix the chinese version in[ FLINK-12944](https://issues.apache.org/jira/projects/FLINK/issues/FLINK-12944?filter=allissues)

Reply:
no, we will fix this link _now_. It is a trivial change and we keep the files in sync.

Reply:
ok， I will fix it in this PR